netplay/createroom.cpp

```diff
@@ -220,3 +241,33 @@ void CreateRoom::handleUseInputDelay(bool useInputDelay)
 {
     inputDelay->setEnabled(useInputDelay);
 }
+
+void CreateRoom::handleServerChanged(int index)
+{
+    if (serverChooser->itemData(index) == "Custom") {
+        bool ok;
+        QString host = QInputDialog::getText(0, "Custom Netplay Server", "IP Address / Host:", QLineEdit::Normal, "", &ok);
```

Comment:
You can probably forget my previous feedback, I'm ok with the timeout. For this call (QInputDialog::getText), the first argument is the parent, so it should be ```this``` instead of 0